packages/components/src/Spinner/Spinner.mdx

```diff
@@ -0,0 +1,53 @@
+---
+name: Spinner
+menu: Components
+route: /components/spinner
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Spinner } from ".";
+import { Text } from "../Text";
+
+# Spinner
+
+<ComponentStatus stage="pre" responsive="no" accessible="yes" />
+
+Spinners are used to visualize an undetermined load time for a section of a page
+and prevent double clicking of a UI element by overlaying it while its action is
+processing.
```

Comment:
I don't feel that it is the responsibility for the spinner to prevent the double clicking. A Spinner is just the UI element.

Reply:
@eddysims this is based off of the SG-1 description, where by default the Spinner element overlays all content it wraps

Reply:
^ but I think to that point, this version of Spinner does not appear to wrap children in the same way, so it is extra true that it can be removed from the description!